### PR TITLE
adding chunking and simplify into defs

### PR DESCRIPTION
**Pull request by sternp**

### CODE REVIEW

File: bin/mqsub [hunk 1]
```diff
@@ -142,6 +282,9 @@ if __name__ == '__main__':
     parser.add_argument('--script-tmpdir', help="When '--script -' is specified, write the script to this location as a temporary file", default='/lustre/scratch/microbiome/tmp')
     parser.add_argument('--poll-interval', help="Poll the PBS server once every this many seconds [default: 30]", type=int, default=30)
     parser.add_argument('--no-executable-check', help="Usually mqsub checks the executable is currently available. Don't do this [default: do check]",action='store_true')
+    parser.add_argument('--file', help="A file with list of newline separated commands to be split into chunks and submitted. One command per line. mqsub --file <file.txt> --chunk-num <int>")
```
Comment (wwood) [inline bin/mqsub:285]: previous comment applies here still re name of this argument

**Review by wwood — COMMENTED** [review]
Hey, thanks, much better. Annoying to properly review given the whitespace changes but I guess you have tested it. Just that one issue from last time.

File: bin/mqsub (hunk 1 above)
Comment (sternp) [reply bin/mqsub:285]: Fair enough. Will update this.

**Comment by sternp** [comment]
Yes, sorry about the whitespace changes. The whole script is basically restructured. Yes, it's been tested so should be good to go if you're happy with that.

**Comment by wwood** [comment]
Thanks. Feel free to merge after.

**Pull request merged**